src/Dashboard/Resources/Filter/Filter.tsx

```diff
@@ -1,99 +1,114 @@
-import React, { useState, useEffect } from 'react';
+import { useState, useEffect } from 'react';
 
 interface Props {
-  onUpdate: (tags: string[]) => void;
+  onUpdate: (tags: string[], count: {'unit': number, 'type': number}) => void;
 }
 
-const Filter = ({ onUpdate }: Props) => {
+let prevTags = [''];
```

Comment:
Since this is a global variable it interferes with renders — can you convert it to a state variable? See this screenshot — the SearchBar is re-rendered in a flow where I just clicked on the checkboxes

<img width="1728" alt="Screenshot 2023-05-09 at 10 40 55 AM" src="https://github.com/parthsarin/stanford-craft/assets/5355724/c45abf5d-5bbe-49e2-ba8b-7469949c621a">